src/main/java/com/team4/testingsystem/security/CustomUserDetails.java

```diff
@@ -31,7 +33,7 @@ public String getPassword() {
 
     @Override
     public String getUsername() {
-        return username;
+        return login;
```

Comment:
Why getUserName returns login?

Reply:
`getUsername()` method is overrided from `UserDetails`, so name cannot be changed.
I used `login` instead of `username` just not to mistake `username` for `name`.
Should I change `login` back to `username`?

Reply:
Good point. Leave it as is